test/functionalTest/cases/1494_subscription_operations/sub_operation_entity_change.test

```diff
@@ -0,0 +1,205 @@
+# Copyright 2022 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Subscription operations entityChange
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create subscription for entity type T with entityChange
+# 02. Create entity E/T (no notif)
+# 03. Update entity E/T without actual change (no notif)
+# 04. Update entity E/T with actual change (notif)
+# 05. Delete entity E/T (no notif)
+# 06. Dump accumulator: see E/T notification with entityStatus entityChange
+#
+
+echo "01. Create subscription for entity type T with entityChange"
+echo "==========================================================="
+payload='{
+  "subject": {
+    "entities": [
+      {
+        "idPattern" : ".*",
+        "type": "T"
+      }
+    ],
+    "condition": {
+      "operations": [ "entityChange" ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://127.0.0.1:'${LISTENER_PORT}'/notify"
+    },
+    "attrs": [ "entityStatus", "*" ]
```

Comment:
As alternantives to entityStatus, it has been suggested:

* triggeredBy
* notifTriggeredBy

Reply:
NTC in this PR. See https://github.com/telefonicaid/fiware-orion/issues/1494#issuecomment-1076235243